ecms/src/main/java/org/exoplatform/platform/qa/ui/ecms/pageobject/SiteExplorerHome.java

```diff
@@ -171,7 +177,10 @@ public void deleteData(String title, boolean... destination) {
 
     info("Right click on nodename");
     ELEMENT_CONTENT_LIST.find(byLinkText(title)).click();
+    if( $(ELEMENT_ACTIONBAR_MORE).is(Condition.visible))
     $(ELEMENT_ACTIONBAR_MORE).click();
+    else
+      $(ELEMENT_ACCOUNT_NAME_LINK).click();
```

Comment:
Missing brackets and indentation

Reply:
We can delete this if because it was a hack to fix fast click; it is not accuarate anymore